src/gui/tray/UserLine.qml

```diff
@@ -25,6 +29,10 @@ MenuItem {
                 hoverEnabled: true
                 flat: true
 
+                Accessible.role: Accessible.Button
+                Accessible.name: name

```

Comment:
This could instead be labeled as "Switch to 'account name'" without description?

Or @tyrylu @Adriorjalesvidal would you say this way works better, with Accessible.name set to the account name and Accessible.description set to "Account button, selecting the corresponding account as active when clicked"?